test/integration/admin/api/services_controller_test.rb

```diff
@@ -96,6 +96,16 @@ class TenantHostTest < ActionDispatch::IntegrationTest
       assert_correct_params
     end
 
+    test 'a member user cannot create a service' do
+      member = FactoryBot.create(:member, account: provider)
+      member_access_token_value = FactoryBot.create(:access_token, owner: member, scopes: %w[account_management], permission: 'rw').value
+
+      assert_no_difference(provider_services.method(:count)) do
+        post admin_api_services_path(access_token: member_access_token_value, format: :json), permitted_params.merge(forbidden_params)
```

Comment:
```suggestion
        post admin_api_services_path(access_token: member_access_token_value, format: :json), permitted_params
```

Reply:
Why is that if I may ask?

Reply:
I agree with @guicassolato 

The purpose of `forbidden_params` is to see that they are filtered out and not created/updated from the params although the `permitted_params` are updated/created. But that is only for successful response codes.
For a `forbidden` response, nothing should be updated/created. Therefore it is confusing adding the `forbidden_params` as if we expected something different to happen with the different kind of params.

Reply:
> You're absolutely right, I don't know how I missed that